tests/lit-tests/1747.ispc

```diff
@@ -0,0 +1,4 @@
+// RUN: %{ispc} --target=sse4-i32x8 %s > %t 2>&1
+// REQUIRES: X86_ENABLED
```

Comment:
This is not specific for x86, so use ``--target=host``, this is the current way to run on both ARM and x86. We may change it, but for now just use host.

Reply:
Done.